src/internal/FluencePeer.ts

```diff
@@ -0,0 +1,350 @@
+import { AirInterpreter, CallServiceResult, LogLevel, ParticleHandler, SecurityTetraplet } from '@fluencelabs/avm';
+import log from 'loglevel';
+import { Multiaddr as MA } from 'multiaddr';
+import PeerId, { isPeerId } from 'peer-id';
+import { CallServiceHandler } from './CallServiceHandler';
+import { PeerIdB58 } from './commonTypes';
+import makeDefaultClientHandler from './defaultClientHandler';
+import { FluenceConnection, FluenceConnectionOptions } from './FluenceConnection';
+import { logParticle, Particle } from './particle';
+import { randomPeerId, peerIdFromEd25519SK } from './peerIdUtils';
+import { RequestFlow } from './RequestFlow';
+import { loadRelayFn, loadVariablesService } from './RequestFlowBuilder';
+import { createInterpreter } from './utils';
+
+/**
+ * Node of the Fluence detwork specified as a pair of node's multiaddr and it's peer id
+ */
+type Node = {
+    peerId: PeerIdB58;
+    multiaddr: string;
+};
+
+/**
+ * Represents all the possible types which can used to specify the connection point. Can be in the form of:
+ * * string - multiaddr in string format
+ * * Multiaddr - multiaddr object, @see https://github.com/multiformats/js-multiaddr
+ * * Node - node structure, @see Node
+ */
+export type ConnectionSpec = string | MA | Node;
+
+/**
+ * Enum representing the log level used in Aqua VM.
+ * Possible values: 'info', 'trace', 'debug', 'info', 'warn', 'error', 'off';
+ */
+export type AvmLoglevel = LogLevel;
+
+/**
+ * Configuration used when initiating Fluence Peer
+ */
+export interface PeerConfig {
+    /**
+     * Node in Fluence network to connect to.
+     * If not specified the will work locally and would not be able to send or receive particles.
+     */
+    connectTo?: ConnectionSpec | Array<ConnectionSpec>;
```

Comment:
Why is it an Array?

Reply:
We will have multiple relays, so there should be a way  to connect to several of them